examples/js/loaders/AssimpJSONLoader.js

```diff
@@ -152,7 +152,7 @@ THREE.AssimpJSONLoader.prototype = {
 
 				function makeColor(start) {
 					var col = new THREE.Color( );
-					col.setRGB( arr[0], arr[1], arr[2] );
+					col.setRGB( start[0], start[1], start[2] );
```

Comment:
I think a better solution would be to leave it as-is and just change the function signature:

    function makeColor( arr ) {

A better solution would be to remove the function and make use of the method `Color.fromArray( array )`.